pages/platform/databases/databases_12_operator/guide.en-gb.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Getting started with database kubernetes opdrator
+excerpt: Find out how to install and use the kubernetes database operator 
+slug: database-operator
+section: General guides
+order: 120
+updated: 2023-05-02
+---
+
+**Last updated May 2<sup>nd</sup>, 2023**
+
+## Objective
+
+The kubernetes database operator allow you to automaticaly authorize your Kubernetes cluster IP on your OVHcloud cloud databases service.
+
+**This guide explains how to install and use the database operator in order to authorize the Kubernetes IP on your service**
+
+## Requirements
+
+- Access to the [OVHcloud API](https://api.ovh.com/) (create your credentials by consulting [this guide](https://docs.ovh.com/gb/en/api/first-steps-with-ovh-api/))
+- A [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
+
+## Ovh Credentials
+The operator needs a secret that contains the credentials to call Ovhcloud api. Go to https://api.ovh.com/createToken/ to generate the credentials namely:
+- application key
+- application secret
+- consumer key
+
+## Values
+Create a values.yaml to be injected in the helm chart 
+that will be created afterwards. Region is either: ovh-eu, ovh-ca or ovh-us.
+```yaml
+ovhCredentials:
+  applicationKey: XXXX
+  applicationSecret: XXXX
+  consumerKey: XXXX
+  region: XXXX # ovh-eu, ovh-ca or ovh-us
+
+namespace: XXXX # Your Kubernetes namespace
+```
+
+## Installation
+Use the kubernetes package manager [helm](https://helm.sh) and the values file you created to install the operator.
+
+```bash
+helm install -f values.yaml public-cloud-databases-operator oci://registry-1.docker.io/ovhcom/public-cloud-databases-operator --version 3
```

Comment:
By the way, instead of asking the user to create a `values.yaml` file you can simply set the parameters in the helm install command line.

Example:

```
$ helm install prometheus-community/kube-prometheus-stack \
--create-namespace --namespace prometheus \
--generate-name \
--values /tmp/kube-prometheus-stack.values \
--set prometheus.service.type=LoadBalancer \
--set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false \
--set grafana.service.type=LoadBalancer \
--set grafana.adminpassword=myawesomepassword
```

Reply:
This one is specially important to me. We should strive to make our open source operators as close as possible to community standards and good practices as possible. A great majority of open source projets use the *Helm chat in Helm repository* approach. It's the market standard, see the [Bitnami list of Helm charts](https://bitnami.com/stacks/helm). Why would we choose to publish in another way?

Specially when we are suggesting that to our own customers using our OVHcloud Managed Registry.

As @scraly said, it's not only the market standard, it also allows to manage the charts in a git repository, version them, and automate the process.

Reply:
In most of the tutorials in our docs we are suggesting adding the parameters in the Helm command line instead of using the `values.yml` file option. 

It could be nice if we keep the same approach here for coherency reasons...